network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -72,7 +73,7 @@ <h3 class="h3-heading h3-heading-small">What does it know about me?</h3>
         {% else %}
           {% criterion "privacy_policy_reading_level" "Privacy policy reading level" product.privacy_policy_reading_level product.privacy_policy_helptext %}
         {% endif %}
-        {% criterion "share_data" "Shares your information with third parties" product.share_data product.share_data_helptext %}
+        {% criterion "share_data" "Shares your information with 3rd parties for unexpected reasons" product.share_data product.share_data_helptext %}
```

Comment:
Why this change? Shareprogress text length limitation? (as a number below 20, normally this should stay "third").

Reply:
not sure but this text is what we have in the ticket /cc @kristinashu

Reply:
This is not related to Shareprogess. This is one of the criteria. Please use "3rd" to help shorten the text. This has been approved by Jen and Abigail the lawyer.